src/rpdk/core/data/managed-upload-infrastructure.yaml

```diff
@@ -117,13 +117,19 @@ Resources:
           Statement:
           - Effect: Allow
             Action:
+            - kms:Encrypt
+            - kms:Decrypt
```

Comment:
This seems particularly dangerous to leave wide open. Can we scope it to just the KMS key above?

Reply:
Good comment. We should have separate policy for KMS and S3, and scope down to the Key and bucket created above.